backend/api-gateway/docker-compose.api-gateway.yml

```diff
@@ -0,0 +1,15 @@
+version: '3.8'
```

Comment:
the docker-compose at project root is version 3.9. Will this different version cause issues?

Reply:
this will not have any issue, the project level yml file is use to automate the service at project level.